tests/integration/cli/classification/test_classification.py

```diff
@@ -110,6 +120,16 @@ def test_otx_train(self, template, tmp_dir_path):
         args1["--load-weights"] = f"{template_work_dir}/trained_{template.model_template_id}/weights.pth"
         otx_train_testing(template, tmp_dir_path, otx_dir, args1)
 
+    @e2e_pytest_component
+    @pytest.mark.parametrize("template", templates, ids=templates_ids)
```

Comment:
how abouting adding `@pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")` in this line?
I think this test don't have to be run in stability tests.

Reply:
Reflected. Thank you.